portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Kayla Leung</title>
+
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font.css">
+
+    <script src="blogScript.js"></script>
+    <script src="https://kit.fontawesome.com/b99e675b6e.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+  </head>
+  <body>
+
+    <!-- Toggle Side View -->
+    <input type="checkbox" name="" \>
+    <span class="icon"><i class="fas fa-bars" alt="menu icon"></i></span>
+    
+    <!-- Left Side Bar -->
+    <div class="sidebar">
+
+        <img src="images/kaylaleung.jpeg" width="175" height="175" alt="Photo of Kayla Leung">
+
+        <div>
+          <h2>Kayla Leung</h2>
+          <h4>Software Engineering Intern</h4>
+        </div>
+
+        <div class="social_media">
+          <ul>
+            <li class="facebook-icon"><a href="//www.facebook.com/kaylayleung"><i class="fab fa-facebook-f" alt="facebook icon"></i></a></li>
+            <li class="instagram-icon"><a href="//www.instagram.com/kaylaleung/"><i class="fab fa-instagram" alt="instagram icon"></i></a></li>
+            <li class="linkedin-icon"><a href="//www.linkedin.com/in/kaylaleung/"><i class="fab fa-linkedin" alt="linkedin icon"></i></a></li>
+            <li class="github-icon"><a href="//www.github.com/kaylaleung"><i class="fab fa-github" alt="github icon"></i></a></li>
+          </ul>
+        </div>
+
+        <div class="menu">
+          <ul>
+              <li class="tab"><a href="/index.html#home"><i class="fas fa-home"></i>Home</a></li>
+              <li class="tab"><a href="/index.html#education"><i class="fas fa-user"></i>Education</a></li>
+              <li class="tab"><a href="/index.html#experience"><i class="fas fa-address-card"></i>Experience</a></li>
+              <li class="tab"><a href="/index.html#projects"><i class="fas fa-project-diagram"></i>Projects</a></li>
+              <li class="tab"><a href="/index.html#blog" class="active"><i class="fas fa-blog"></i>Blog</a></li>
+          </ul>
+        </div>
+    </div>
+
+    <!-- Main Content Section -->
+
+    <div class="menu">
+      <section class="banner">
+        <div class="info">
+          <div class ="title"><h2>Blog Title</h2></div>
+          
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
Whitespace/indentation here seems a bit off

Reply:
I think this is a weird cloud shell thing because it looks fine on my end, but then the commit still looks wonky. If anything I think the next PR using vscode should help fix it.